### PR TITLE
add ExecuteQueryOnHiPerfClusterWithBackup to KustoDataProvider

### PR DESCRIPTION
**Pull request by yifguoMSFT**

### CODE REVIEW

File: src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs [hunk 1]
```diff
@@ -288,5 +288,18 @@ public async Task<string> GetAggHiPerfClusterNameByStampAsync(string stampName)
                 return null;
             }
         }
+
+        public async Task<DataTable> ExecuteQueryOnHiPerfClusterWithBackup(string aggQuery, string backupQuery, string stampName, string requestId = null, string operationName = null) 
+        {
+            string cluster = await GetAggHiPerfClusterNameByStampAsync(stampName);
+            if (cluster == null)
```
Comment (nmallick1) [inline src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs:295]: nit: Can we do !string.IsNullOrWhiteSpace(cluster)?

```suggestion
            if (!string.IsNullOrWhiteSpace(cluster))

```

File: src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs [hunk 2]
```diff
@@ -288,5 +288,18 @@ public async Task<string> GetAggHiPerfClusterNameByStampAsync(string stampName)
                 return null;
             }
         }
+
+        public async Task<DataTable> ExecuteQueryOnHiPerfClusterWithBackup(string aggQuery, string backupQuery, string stampName, string requestId = null, string operationName = null)
```
Comment (nmallick1) [inline src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs:292]:
```suggestion
        public async Task<DataTable> ExecuteQueryOnHiPerfClusterWithBackup(string aggQuery, string fallbackQuery, string stampName, string requestId = null, string operationName = null) 
```

File: src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs (hunk 1 above)
Comment (yifguoMSFT) [reply src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs]: sure, fixe4d

File: src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs (hunk 2 above)
Comment (yifguoMSFT) [reply src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs]: updated

**Review by nmallick1 — APPROVED** [review]

**Pull request merged**